code/modules/surgery/organs/external/wings/wings.dm

```diff
@@ -11,6 +11,14 @@
 
 	organ_flags = parent_type::organ_flags | ORGAN_EXTERNAL
 
+/obj/item/organ/wings/on_mob_insert(mob/living/carbon/organ_owner, special, movement_flags)
+	. = ..()
+	add_organ_trait(TRAIT_SOFT_FALL)
+
+/obj/item/organ/wings/on_mob_remove(mob/living/carbon/organ_owner, special, movement_flags)
+	. = ..()
+	remove_organ_trait(TRAIT_SOFT_FALL)
+
```

Comment:
These should just be in the `organ_traits` list

Also, moth wings need to `remove_organ_trait(TRAIT_SOFT_FALL)` when burnt. Your PR currently breaks that feature